react/src/components/ResourceGroupList.tsx

```diff
@@ -132,17 +172,85 @@ const ResourceGroupList: React.FC = () => {
                 setOpenCreateModal(true);
               }}
             />
-            <Button
-              type="text"
-              size="large"
-              icon={
-                <DeleteOutlined
-                  style={{
-                    color: token.colorError,
-                  }}
-                />
+            <Tooltip
+              title={
+                record.is_active
+                  ? t('resourceGroup.Deactivate')
+                  : t('resourceGroup.Activate')
               }
-            />
+            >
+              <Popconfirm
+                title={
+                  record.is_active
+                    ? t('resourceGroup.DeactivateResourceGroup')
+                    : t('resourceGroup.ActivateResourceGroup')
+                }
+                placement="left"
+                okType={record.is_active ? 'danger' : 'primary'}
+                okText={
+                  record.is_active
+                    ? t('resourceGroup.Deactivate')
+                    : t('resourceGroup.Activate')
+                }
+                description={record?.name}
+                onConfirm={() => {
+                  commitUpdateResourceGroup({
+                    variables: {
+                      name: record.name ?? '',
+                      input: {
+                        is_active: !(activeType === 'active'),
```

Comment:
```suggestion
                        is_active: !record.is_active,
```
I think it's more clearer.